secrets.auto.tfvars.sample

```diff
@@ -0,0 +1,2 @@
+infura_api_key = "<located_in_...>"
```

Comment:
Do we have a place where devs can access these values in a secure manner?

Reply:
The infura values? Or tf secrets?

Reply:
All of them. I ll just point in the comment where to look for each of them specifically 'til we come up with a solution.